Apps/WebClient/src/ClientApp/src/components/timeline/linearTimeline.vue

```diff
@@ -120,9 +120,6 @@ export default class LinearTimelineComponent extends Vue {
     private onVisibleEntriesUpdate() {
         if (this.visibleTimelineEntries.length > 0) {
             if (this.isVisible) {
-                console.log(
-                    "notifying calendar vue about timeline's page updated..."
-                );
```

Comment:
Are we not recommended to write console.log? What are the down side of having more logs on our console?

Reply:
Too many logs would clutter the console. We have a story about improving the front end logs.

Reply:
so should we leave this remove as a part of that improving the front end logs then as this PR should not be related to this cleanup exercise right?
Also, how often this event gets triggered to clutter the console? I'm seeing issue with not enough logging more often then a lot of logs.

Reply:
We can add them better once we have that improved framework. Regarding this one I removed since it was happening every single time the page changed.

Reply:
I know it logs every time when page changed which is an important event to know when it happens. For important/key events, I believe we should log them. But I'm good with waiting for the logging framework.

Reply:
not sure as I think it is better to log all the important events, unless you think the page changed is not an important event. We should review the art of logging, what needs to be logged and what not. Currently if we are not sure, just leave code as is right unless you want to suggest us not to log important event

Reply:
Its removed

Reply:
why not waiting for the framework discussion/implementation to make this call? Is this remove necessary as we don't even know if we should or if it is good remove these logs or not for now right?